feature/CourseView/Sources/CourseView/ExerciseTab/ExerciseDetailView.swift

```diff
@@ -70,31 +70,12 @@ public struct ExerciseDetailView: View {
     public var body: some View {
         DataStateView(data: $exercise, retryHandler: { await loadExercise() }) { exercise in
             ScrollView {
-                VStack(alignment: .leading, spacing: .m) {
-                    VStack(alignment: .leading, spacing: .m) {
-                        ForEach(exercise.baseExercise.categories ?? [], id: \.category) { category in
-                            Chip(text: category.category, backgroundColor: UIColor(hexString: category.colorCode).suColor)
-                        }
-                        if let dueDate = exercise.baseExercise.dueDate {
-                            Text(R.string.localizable.dueDate(dueDate.relative ?? "?"))
-                        } else {
-                            Text(R.string.localizable.noDueDate())
-                        }
-                        HStack {
-                            Text(R.string.localizable.points(
-                                score,
-                                exercise.baseExercise.maxPoints?.clean ?? "0"))
-                            if exercise.baseExercise.includedInOverallScore != .includedCompletly {
-                                Chip(text: exercise.baseExercise.includedInOverallScore.description, backgroundColor: exercise.baseExercise.includedInOverallScore.color)
-                            }
-                            if let assessmentType = exercise.baseExercise.assessmentType?.description {
-                                Text(R.string.localizable.assessment(assessmentType))
-                            }
-                        }
-                        SubmissionResultStatusView(exercise: exercise)
-                        if let latestResultId,
-                           let participationId,
-                           showFeedbackButton {
+                VStack(alignment: .leading, spacing: .l) {
+                    // HStack for all buttons regarding viewing feedback and for the future, starting an exercise
+                    if let latestResultId,
+                       let participationId,
+                       showFeedbackButton {
```

Comment:
I would put this in one line.

(Currently there is only a warning at a line length of 200, but I have a page guide at 120 in Xcode)